src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -755,6 +755,12 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t, uct_iface_ops_t *tl_ops,
         return UCS_ERR_INVALID_PARAM;
     }
 
+    if (mlx5_config->tm.seg_size > UCT_IB_MLX5_MP_RQ_BYTE_CNT_MASK) {
+        ucs_error("TM segment size is too big %ld, it must not exceed %d",
+                  mlx5_config->tm.seg_size, UCT_IB_MLX5_MP_RQ_BYTE_CNT_MASK);
+        return UCS_ERR_INVALID_PARAM;
```

Comment:
merge artifact?

Reply:
Indeed, sorry for that, should be gone now.